test/express.test.js

```diff
@@ -37,9 +36,85 @@ describe('Server tests', function () {
       }
     };
 
+    const url = 'http://127.0.0.1:8080/api?relStart=1655888897602&relEnd=1655888906678&timeseries=eventloop&timeseries=memory';
+    const response =  await fetch(url);
+    const data = await response.json();
+    
     expect(response.status).to.equal(200);
     expect(response.headers.get('content-type')).to.include('application/json');
     expect(data).to.deep.equal(expectedData);
   });
+
+  it('returns correct data when provided no file', async function() {
+    testServer = new Server(['./servers/express.js','http:127.0.0.1:8080']);
+    await testServer.readyPromise;
+    
+    const response =  await fetch('http://127.0.0.1:8080/api');
+    const data = await response.json();
+    
+    const expectedData = {
+      range: {},
+      timeseries: {
+        eventloop: [],
+        memory: [],
+        cpu: []
+      }
+    };
+
+    expect(response.status).to.equal(200);
+    expect(response.headers.get('content-type')).to.include('application/json');
+    expect(data).to.deep.equal(expectedData);
+  });
+
+  it('only allows text files to be uploaded', async function() {
+    testServer = new Server(['./servers/express.js','http:127.0.0.1:8080']);
+    await testServer.readyPromise;
+    
+    // upload a couple files
+    const form = new FormData();
+    form.append('file', fs.createReadStream(path.join(__dirname, 'sample-data.log')));
+    form.append('file', fs.createReadStream(path.join(__dirname, 'servers', 'express.js')));
+
+    const response =  await fetch('http://127.0.0.1:8080/api/upload', {method: 'POST', body: form});
+    const data = await response.json();
+    
+    // check if everything's right
+    expect(response.status).to.equal(200);
+    expect(data.files.length).to.equal(1);
+    expect(data.files[0].mimetype).to.equal('text/plain');
+
+    // delete any files created during the test
```

Comment:
it might also make sense to put the deletes in an `afterEach()` so that code can be moved out of each test. it may have to glob them first in the directory, but better to do it once.